### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by wizofe**
Update algorithm description with a simple youtube explanation from BBC videos.

**Description**

### CODE REVIEW

**Comment by Bouncey** [comment]
Deploy preview failed.

Built with commit 2bd98c7ea44c489b2e8de0012c264dcd92e9c186

https://app.netlify.com/sites/guide/deploys/5a48689a4c4b935ca60eabf9

**Comment by wizofe** [comment]
Sure, done.

File: src/pages/algorithms/index.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
+----
+-title: Algorithms
```
Comment (Asjas) [inline src/pages/algorithms/index.md]: Please remove the hyphen before the word `title`.

File: src/pages/algorithms/index.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
+----
```
Comment (Asjas) [inline src/pages/algorithms/index.md]: Please make this only 3 hyphens.

File: src/pages/algorithms/index.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+-title: Algorithms
+----
```
Comment (Asjas) [inline src/pages/algorithms/index.md]: Please make this also only 3 hyphens.

**Comment by Asjas** [comment]
Hi @wizofe 

I have added comments to your last commit requesting changes to be made. Your PR is failing the Travis CI build.

You can refer to the [Contributing Guide](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#article-style-guide) to make sure your article conforms to the `Article Style Guide`. Once you have made the necessary changes the Travis build should pass. Once it has passed I will review and merge your PR.

Thank you for your patience.

**Comment by Asjas** [comment]
Hey @wizofe 

Due to no response to the review requested I will be closing this and marking as stale

---

> Feel free to reference the [Article Style Guide](https://github.com/freeCodeCamp/guides#article-title) for this repo on formatting an article correctly so your Travis CI build passes. ✅
>
> Also, it's good practice on GitHub to write a brief description of your changes when creating a PR. 📝
Unfortunately your PR is failing the Travis Build. Please refer to our [Style Guide](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#article-style-guide) on how a article should look.

**Pull request closed without merging**